netutils/codecs/md5.c

```diff
@@ -406,4 +411,59 @@ char *md5_hash(const uint8_t * addr, const size_t len)
   return hash;
 }
 
+/****************************************************************************
+ * Name: md5_file
+ *
+ * Description:
+ *   MD5 hash for a file
+ *
+ * Input Parameters:
+ *   path: File Path
+ *   mac : Buffer for the hash
+ *
+ ****************************************************************************/
+
+int md5_file(const char *path, uint8_t *mac)
```

Comment:
General comment: Need to add `FAR` to pointers

Reply:
@pkarashchenko The whole code base doesn't add FAR, so it's better to put into the new patch.

Reply:
Yes. Sure. It is more like a "note" comment